tests/core/RequestHandler.spec.ts

```diff
@@ -130,6 +136,30 @@ describe('RequestHandler', () => {
     await batchScheduler.writeOperationBatch();
   });
 
+  it('should process create operation from test vectors correctly', async () => {
+    const createOperationBuffer = Buffer.from(JSON.stringify(createFixture));
+    const response = await requestHandler.handleOperationRequest(createOperationBuffer);
+    expect(response.status).toEqual(ResponseStatus.Succeeded);
+  });
+
+  it('should process update operation from test vectors correctly', async () => {
+    const updateOperationBuffer = Buffer.from(JSON.stringify(updateFixture));
+    const response = await requestHandler.handleOperationRequest(updateOperationBuffer);
+    expect(response.status).toEqual(ResponseStatus.Succeeded);
```

Comment:
good start, but perhaps we should assert the expected did document here?... otherwise we are just testing the the operation was accepted... we are not testing that the did document was properly constructed from it.